modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,99 +82,562 @@ The following charts are available only if you are using Unified Agent and have
+
+Outbound - Total Requests by Response Type::
+The sum of outbound requests, grouped by type of response in automatic intervals. Responses are OK or Failed. If a Mule app has more than one connector type, all types are included in this metric.
+
+Outbound - Average Response Time::
+The average response time of outbound requests, grouped by application in automatic intervals. If a Mule app has more than one connector type, all types are included in this metric.
+
+Outbound - Response Time by HTTP Endpoint::
+The average, maximum, and minimum and total response time, grouped by endpoint.
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Outbound - Total Requests by HTTP Endpoint::
+The sum of outbound requests in the period, grouped by endpoint.
+
+Outbound - Response Time (99, 90, 75, and 50 percentile)::
+The average response time of the 99, 90, 75, and 50 percentiles of outbound requests, grouped by application in automatic intervals. The unit for these metrics is milliseconds (ms).
+
+Outbound - Failure Total Request Count::
+The sum of failed outbound requests, grouped by type of response in automatic intervals.
+
+Outbound - Ports::
+The sum of outbound requests, grouped by remote port in automatic intervals.
+
+// ports?
+
+Outbound - Operations::
+The sum of outbound requests, grouped by operation in automatic intervals.
+
+Outbound - Protocols::
+The sum of outbound requests, grouped by protocol in automatic intervals.
+
+Outbound - Endpoints::
+The sum of outbound requests, grouped by remote hosts (endpoints) in automatic intervals.
+
+[[flows-charts]]
+== Flows
+
+The following charts appear on the *Flows* dashboard:
+
+Inbound Response Time by Flow::
+The average inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average outbound response time, grouped by flow.
+
+Inbound Requests by Flow::
+The sum of inbound requests, grouped by flow.
+
+Outbound Requests by Flow::
+The sum of outbound requests, grouped by flow.
+
+Failures by Flow::
+The sum of failed inbound requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The sum of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The sum of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Inbound Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
```

Comment:
```suggestion
The average, minimum, and maximum inbound response times, grouped by flow.
```

Reply:
GLOBAL
I also just noticed that these definitions are fragments, which do not require periods...unless one or more items in a single list includes a complete sentence, in which case all take periods.